service/javadsl/kafka/server/src/test/scala/com/lightbend/lagom/internal/javadsl/broker/kafka/JavadslKafkaApiSpec.scala

```diff
@@ -394,7 +399,7 @@ object VarianceCompat {
 object JavadslKafkaApiSpec {
   private val test1EventJournal = new EventJournal[String]
   private val test2EventJournal = new EventJournal[String]
-  private val test3EventJournal = new EventJournal[String]
+  private val test3EventJournal = new EventJournal[(String, Boolean)]
   private val test4EventJournal = new EventJournal[String]
   private val test5EventJournal = new EventJournal[String]
   private val test6EventJournal = new EventJournal[String]
```

Comment:
If we're pushing forward with the suport for multimessage in this PR we will need a few tests for it.

Reply:
reverted